assets/js/views/ChargingSessions.vue

```diff
@@ -12,108 +12,107 @@
 
 		<div class="row">
 			<main class="col-12">
-				<div class="mb-4">
+				<div class="d-flex justify-content-start mb-4">
 					<a
 						class="btn btn-outline-secondary text-nowrap my-2"
-						:href="`./api/sessions?format=csv&amp;lang=${$i18n.locale}`"
+						:href="csvHrefLink($i18n.locale)"
 						download="sessions.csv"
 					>
 						{{ $t("sessions.downloadCsv") }}
 					</a>
 				</div>
 
 				<div v-for="group in sessionsByMonthAndLoadpoint" :key="group.month">
-					<div class="mx-2">
-						<div class="d-flex align-items-baseline my-5">
-							<h2 class="me-4 mb-0">
-								{{ formatGroupHeadline(group.month) }}
-							</h2>
-						</div>
+					<div class="d-flex align-items-baseline my-5">
+						<h2 class="me-4 mb-0">
+							{{ formatGroupHeadline(group.month) }}
+						</h2>
+						<a
+							class="btn btn-sm btn-outline-secondary text-nowrap my-2"
```

Comment:
`my-2` sollte hier keinen Effekt haben. Das collapsed mit dem `my-5` des Elternelements. Kann also raus.

Optisch gefällt mir noch nicht so gut, wie sich die Überschrift zu dem Button verhält. Der Text von beiden hat die gleiche Baseline, das ist gut, trotzdem wirkt das unrund. Einen kleineren Button hat Bootstrap aktuell leider nicht mehr. Früher gabs mal `btn-xs`. Vielleicht führen wir den für eins einfach wieder ein. Ich hab mal etwas im Browser rumgespielt:

<img width="377" alt="Bildschirm­foto 2023-01-23 um 07 52 02" src="https://user-images.githubusercontent.com/152287/213981654-8e6819a1-71a8-47b4-ad77-03fd136ef5d0.png">

```
.btn-xs {
    --bs-btn-padding-y: 0;
    --bs-btn-padding-x: 0.5rem;
    --bs-btn-font-size: 0.70rem;
    --bs-btn-border-radius: 0.25rem;
    --bs-btn-line-height: 2;
}
```

Finde ich runder. Du könntest den Code in die `app.css` global mit aufnehmen.